core/src/banking_stage/latest_unprocessed_votes.rs

```diff
@@ -845,7 +846,12 @@ mod tests {
     #[test]
     fn test_forwardable_packets() {
         let latest_unprocessed_votes = LatestUnprocessedVotes::new();
-        let bank = Arc::new(Bank::default_for_tests());
+        let mut bank = Bank::default_for_tests();
+        bank.set_epoch_stakes_for_test(
+            bank.epoch().saturating_add(1),
+            EpochStakes::new_for_tests(HashMap::new(), bank.epoch().saturating_add(1)),
+        );
```

Comment:
Can't these be embedded in `Bank::default_for_tests()`?

Reply:
The test helpers for bank are a clown show (among other things see new_for_tests_with_config calling new_with_config_for_tests) - I didn't want to get sidetracked into fixing that mess.